examples/date_binding.h

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include "../dev/functional/cxx_core_features.h"
+
+// need support for #include <compare> for advanced std::chrono members
+#if defined(SQLITE_ORM_THREE_WAY_COMPARISON_SUPPORTED)
+
+#include <chrono>
+#include <sstream>
+#include <regex>
+#include <format>
+
+inline std::chrono::sys_days today() {
```

Comment:
I'd move this code right inside example file (chrono_binding.h) just like it is done in `enum_binding.h` to make example self-sufficient

Reply:
I understand but on the other hand it is good to have the binding in a separate header which is self-sufficient so people can just #include it if they want to have a basic implementation of sys_days binding.. What do you think about this?

Reply:
> I understand but on the other hand it is good to have the binding in a separate header which is self-sufficient so people can just #include it if they want to have a basic implementation of sys_days binding.. What do you think about this?

People will not be able to include it cause it's not located in include dir. This header is located inside `tests` project which isn't available in public actually. And that's right cause example is not a thing to include

Reply:
I have placed the header in examples folder - you are telling me that examples is not available to public too?
but its ok if it is not in the #include folder... it still can be copy paste the .h file to a folder of convenience to the developer
In this case the binding example IS a thing that can be included and we immediately have support for sys_days

Reply:
the example is the CPP file

Reply:
> you are telling me that examples is not available to public too?

If you install the lib with any package manager then yes, you'll get only `include`(`dev`) contents, that's all.

>  it still can be copy paste the .h file to a folder of convenience to the developer

if this code is designed to be copied and pasted it can be copied and pasted from `.cpp` file the same way. I don't see any reason of having one example in two files

Reply:
the reusable and self-sufficient is the date_binding.h file

Reply:
why do you think so? It has custom date to string converter. Different developers use different formats to serialize data to string. Some of them even serialize date to int as unix time (like me e.g.)

Reply:
ok, not too important, it's okay will merge the 2 files

Reply:
thank you!

Reply:
you are very welcome !!! I truly respect your opinions, BTW :-)

Reply:
This header file still exists?

Reply:
no. Just a comment